src/peft/utils/__init__.py

```diff
@@ -49,6 +49,7 @@
     _is_valid_match,
     infer_device,
     get_auto_gptq_quant_linear,
+    get_gptqmodel_quant_linear,
```

Comment:
Let's add this function to `__all__` below. This should make the linter happy.

Reply:
Done.